### DISCUSSION TITLE
Slow file transfer windows to windows

### DISCUSSION THREAD

**Discussion by skoky** [rustdesk/rustdesk-server-pro#490] [General]
I am experiencing very slow data transfer while copying files from Windows machine to another Windows machine. I have tested this on several machines. If I do the same from Mac to Windows, the data transfer is much more faster, about 10 times.
My environment:
- Windows machine Version: 1.3.5
- Mac version: 1.2.3  (I will try to upgrade to latest and let you know)
- server version Welcome to RustDesk Server Pro 1.4.6 (self hosted)

**Comment by rustdesk** [comment]
@fufesou

**Comment by fufesou** [comment]
@skoky Hi, did you use the directly `copy&paste` in the remote control (win->win) or the file transfer connection window?

![image](https://github.com/user-attachments/assets/755ff2e9-fbfb-48e4-8523-e5957ce7637e)

Could you please also show the connection quality monitor?

![image](https://github.com/user-attachments/assets/cd3b59ae-23ce-434a-be50-945ffb518ab0)

**Reply by fufesou** [reply]
@skoky Hi, how do you copy files from Windows to Windows? Directly `Ctrl+C/V` or use the file transfer connection(window)?

**Comment by skoky** [comment]
@fufesou I can't replicate the issue anymore. Both transfers from Win-Win and Mac-Win are fast. I used to transfer files using the "Send >" button and the speed difference was very different. It was probably caused by some networking issue. Thanks for your time
